journald/journald.hpp

```diff
@@ -32,6 +37,19 @@ struct Flags : public virtual flags::FlagsBase
       "before it is written to journald.  See '--labels'."
       "\n");
 
+    add(&Flags::destination_type,
+        "destination_type",
+        "Determines where logs should be piped.\n"
+        "Valid destinations include: 'journald', 'sandbox', or 'both'.",
```

Comment:
1. The intention is to not add anymore types to this logger, but to add any future loggers as config changes.

2. 👍